### PR TITLE
Minor review feedback

### PR DESCRIPTION
**Pull request by halter73**
Don't just merge. I added some comments like

> // TODO: Maybe color-code/style the group name, connection id and message differently.

and

> <-- The URL is wrong if VS generates a launchSettings.json file

### CODE REVIEW

File: wwwroot/js/site.js [hunk 1]
```diff
@@ -1,7 +1,5 @@
 ﻿"use strict";
 
-var connectionBuilder = new signalR.HubConnectionBuilder();
```
Comment (halter73) [inline wwwroot/js/site.js]: Previously this was using the same HubConnectionBuilder variable to build both clockConnection and chatConnection within the same page.

While it technically wasn't broken they way it was bing used, HubConnectionBuilder is mutable, so reusing it is a recipe for problems.

File: Pages/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -76,6 +76,9 @@
     <script src="~/js/signalr.min.js"></script>
     <script src="~/js/site.js" asp-append-version="true"></script>
     <script src="~/js/serverTime.js"></script>
+    <script>
+        updateServerTime("@DateTime.UtcNow.ToString("o")");
```
Comment (halter73) [inline Pages/Shared/_Layout.cshtml]: This simply prevents the server time from being blank the first second the page is loaded.

File: Hubs/ChatHub.cs [hunk 3]
```diff
@@ -21,14 +22,9 @@ public async Task AddToGroup(string groupName)
 
         public async Task RemoveFromGroup(string groupName)
         {
-            await Groups.RemoveFromGroupAsync(Context.ConnectionId, groupName);
-
-            await Clients.Group(groupName).SendAsync("Send", $"{Context.ConnectionId} has left the group {groupName}.");
-        }
+            await Clients.Group(groupName).SendAsync("Send", $"{Context.ConnectionId} is leaving the group {groupName}.");
 
-        public Task SendPrivateMessage(string user, string message)
```
Comment (halter73) [inline Hubs/ChatHub.cs]: This wasn't being called anywhere.

File: wwwroot/js/site.js (hunk 1 above)
Comment (bradygaster) [reply wwwroot/js/site.js]: thanks for the info!

**Comment by bradygaster** [comment]
@halter73 thanks for the feedback. i've implemented all of your suggestions in master and didn't merge the PR per your suggestion. closing for now. will templatize and submit to the team for final review by eow.

**Pull request closed without merging**